cppcache/benchmark/GeodeLoggingBM.cpp

```diff
@@ -35,9 +35,9 @@ using apache::geode::client::internal::geode_hash;
 
 const int STRING_ARRAY_LENGTH = 3;
 
-int g_iteration = 0;
+static int g_iteration = 0;
```

Comment:
I'll have to look closer at the benchmark code (we have no instructions in the CONTRIBUTING.md file about running benchmarks which seems bad...)

For the old style integration tests, I don't think I'd worry about the paradigm, since that code is not really getting updated, more just ported to the new style tests.  In the actual production code, I don't think we want to use an anonymous namespace

Reply:
Curious, why wouldn't you want production code to use an anonymous namespace?